src/strokes.ts

```diff
@@ -130,114 +130,87 @@ export class Stroke extends Modifier {
-      case Stroke.Type.BRUSH_UP:
-        arrow = 'arrowheadBlackDown';
-        arrowShiftX = 0.5;
-        arrowY = botY + lineSpace / 2;
-        topY -= lineSpace / 2;
-        break;
       case Stroke.Type.ROLL_DOWN:
       case Stroke.Type.RASQUEDO_DOWN:
-        arrow = 'arrowheadBlackUp';
-        arrowShiftX = -3;
-        textShiftX = this.xShift + arrowShiftX - 2;
-        if (isStaveNote(note)) {
-          topY += 1.5 * lineSpace;
-          if ((botY - topY) % 2 !== 0) {
-            botY += 0.5 * lineSpace;
-          } else {
-            botY += lineSpace;
-          }
-          arrowY = topY - lineSpace;
-          textY = botY + lineSpace + 2;
-        } else {
-          topY += 1.5 * lineSpace;
-          botY += lineSpace;
-          arrowY = topY - 0.75 * lineSpace;
-          textY = botY + 0.25 * lineSpace;
-        }
+        arrow = '\ueb78' /*arrowheadBlackUp*/;
+        arrowY = topY;
+        topY -= lineSpace / 2;
+        botY += lineSpace / 2;
         break;
+      case Stroke.Type.BRUSH_UP:
       case Stroke.Type.ROLL_UP:
       case Stroke.Type.RASQUEDO_UP:
-        arrow = 'arrowheadBlackDown';
-        arrowShiftX = -4;
-        textShiftX = this.xShift + arrowShiftX - 1;
-        if (isStaveNote(note)) {
-          arrowY = lineSpace / 2;
-          topY += 0.5 * lineSpace;
-          if ((botY - topY) % 2 === 0) {
-            botY += lineSpace / 2;
-          }
-          arrowY = botY + 0.5 * lineSpace;
-          textY = topY - 1.25 * lineSpace;
-        } else {
-          topY += 0.25 * lineSpace;
-          botY += 0.5 * lineSpace;
-          arrowY = botY + 0.25 * lineSpace;
-          textY = topY - lineSpace;
-        }
+        arrow = '\ueb7c' /*arrowheadBlackDown*/;
+        arrowY = botY + lineSpace;
+        topY -= lineSpace / 2;
         break;
       case Stroke.Type.ARPEGGIO_DIRECTIONLESS:
-        topY += 0.5 * lineSpace;
-        botY += lineSpace; // * 0.5 can lead to slight underlap instead of overlap sometimes
+        topY -= lineSpace / 2;
+        botY += lineSpace / 2;
         break;
       default:
         throw new RuntimeError('InvalidType', `The stroke type ${this.type} does not exist`);
     }
 
-    // The first letter is capitalized.
-    // We later prepend 'stroke' to create a camelCased glyph category: 'strokeStraight' or 'strokeWiggly'.
-    // See: common_metrics.ts
-    let type = 'Straight';
-
     // Draw the stroke
     if (this.type === Stroke.Type.BRUSH_DOWN || this.type === Stroke.Type.BRUSH_UP) {
       ctx.fillRect(x + this.xShift, topY, 1, botY - topY);
     } else {
-      type = 'Wiggly';
-      if (isStaveNote(note)) {
-        for (let i = topY; i <= botY; i += lineSpace) {
-          Glyph.renderGlyph(ctx, x + this.xShift - 4, i, this.renderOptions.fontScale, 'vexWiggleArpeggioUp');
-        }
+      const lineGlyph =
+        arrow === '\ueb7c' /*arrowheadBlackDown*/ ? '\ueaa9' /*wiggleArpeggiatoUp*/ : '\ueaaa'; /*wiggleArpeggiatoDown*/
```

Comment:
What are we trying to do here? Can we document the reasoning with a comment?

Reply:
Documented and selection changed: down for down and up for up